andela-mugochukwu/DocManager
# Implement feedback

#### What does your PR do?
Redesign search algorithm while implementing feedback
#### Description of Task(s) you completed?
- Implemented every word search algorithm
- Restructured exporting controllers
- Enabled document update feature
#### How should this PR be manually tested?
Start the application and type something in the search box to see the filtered result.
Also view documents with access rights and click on edit to edit it.
#### Any background context you want to provide?
N/A
#### Screenshots (if available)
N/A
#### Questions:
N/A
